mst/dblink-extension.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Using the `dblink` extension in Managed Service for TimescaleDB
+excerpt: Learn how to use `dblink` extension and connect to other PostgreSQL databases
+tags: [extension]
+---
+
+# Use the PostgreSQL `dblink` extension
+
+`dblink` is a [PostgreSQLextension](https://www.postgresql.org/docs/current/dblink.html)
+that allows you to connect to other PostgreSQL databases and to run arbitrary queries.
+
+With [Foreign Data Wrappers](https://www.postgresql.org/docs/current/postgres-fdw.html)(FDW)
+you can define a remote `foreign server` in order to access its data. The
+database connection details such as hostnames are kept in a single place, and you
+only need to create a `user mapping` to store remote connections credentials.
+
+## Before you begin
+
+To create a `dblink` foreign data wrapper you need the following information
+about the PostgreSQL remote server:
+
+*   `HOSTNAME`: The remote database hostname
+*   `PORT`: The remote database port
+*   `USER`: The remote database user to connect. The default user is `tsdbadmin`.
+*   `PASSWORD`: The remote database password for the
+  `USER`
+*   `DATABASE_NAME`: The remote database name. The default database name is 1`defaultdb`.
+
+<highlight type="note">
+The details are available in the [MST portal](https://portal.managed.timescale.com)
+service `Overview` tab or use the `avn service get` command if you installed
+Aiven client.
+</highlight>
+
+<procedure>
+
+### Enable `dblink` extension on MST for PostgreSQL
+
+To enable the `dblink` extension on an MST PostgreSQL service:
+
+1.  Connect to the database as the `tsdbadmin` user:
+
+```bash
+    psql -x "postgres://tsdbadmin:<PASSWORD>@<HOSTNAME>:<PORT>/defaultdb?sslmode=require"
+```
```

Comment:
Indentation for this (and following code blocks) needs to match the list indentation